packages/cryptoplease/lib/presentation/screens/authenticated/send_flow/common/components/enter_address_input_widget.dart

```diff
@@ -29,63 +32,74 @@ class EnterAddressInputWidget extends StatelessWidget {
+          child: DecoratedWindow(
+            isScrollable: false,
+            backgroundStyle: BackgroundStyle.light,
+            hasLogo: false,
+            hasAppBarBorder: false,
+            bottomButton: CpContentPadding(
+              child: CpButton(
+                text: context.l10n.next,
+                minWidth: double.infinity,
+                onPressed: onRecipientSelected,
+              ),
+            ),
+            child: CpContentPadding(
+              child: Column(
+                crossAxisAlignment: CrossAxisAlignment.start,
+                children: [
+                  TextField(
+                    controller: controller,
+                    maxLines: 3,
+                    style: const TextStyle(
+                      fontSize: 18,
+                      fontWeight: FontWeight.w500,
+                      color: CpColors.primaryTextColor,
+                      fontFamily: 'DIN',
                     ),
-                    fillColor: CpColors.lightTextFieldBackgroundColor,
-                    filled: true,
-                    hintText: inputLabel,
-                  ),
-                ),
-                const SizedBox(height: 16),
-                Row(
-                  crossAxisAlignment: CrossAxisAlignment.start,
-                  children: [
-                    CpButton(
-                      text: context.l10n.paste,
-                      size: CpButtonSize.micro,
-                      onPressed: _setFromClipboard,
+                    decoration: InputDecoration(
+                      border: const OutlineInputBorder(
+                        borderRadius: BorderRadius.all(Radius.circular(7)),
+                        borderSide: BorderSide.none,
+                      ),
+                      fillColor: CpColors.lightTextFieldBackgroundColor,
+                      filled: true,
+                      hintText: inputLabel,
                     ),
-                    const SizedBox(width: 8),
-                    CpButton(
-                      text: context.l10n.scanQRCode,
-                      size: CpButtonSize.micro,
-                      onPressed: onQrCodeSelected,
-                    )
-                  ],
-                )
-              ],
+                  ),
+                  const SizedBox(height: 16),
+                  Row(
+                    crossAxisAlignment: CrossAxisAlignment.start,
+                    children: [
+                      CpButton(
+                        text: context.l10n.paste,
+                        size: CpButtonSize.micro,
+                        onPressed: _setFromClipboard,
+                      ),
+                      const SizedBox(width: 8),
+                      CpButton(
+                        text: context.l10n.scanQRCode,
+                        size: CpButtonSize.micro,
+                        onPressed:
+                            context.read<FtSendFlowRouter>().onQrCodeSelected,
```

Comment:
Same here with the tear-off. Besides that, if `onQrCodeSelected` is not used anymore, it should be removed.